include/lsst/jointcal/PhotomModel.h

```diff
@@ -17,23 +17,23 @@ class PhotomModel
 {
 public :
 
-  //! Assign indices to parameters involved in mappings, starting at FirstIndex. Returns the highest assigned index.
-  virtual unsigned AssignIndices(const std::string &WhatToFit, unsigned FirstIndex) = 0;
+  //! Assign indices to parameters involved in mappings, starting at FirstIndex.
+  //! Returns the highest assigned index.
+  virtual unsigned assignIndices(const std::string &whatToFit, unsigned firstIndex) = 0;
 
   //! Offset the parameters by the provided amounts.
-  /*! The shifts are applied according to the indices given in
-      AssignIndices. */
-  virtual void OffsetParams(const Eigen::VectorXd &Delta) = 0;
+  //! The shifts are applied according to the indices given in
+  //! AssignIndices.
```

Comment:
I fixed all the `assignIndices` namings, but I can't really help with the docstring, other than convert it to doxygen.